accounts/main.go

```diff
@@ -38,6 +39,14 @@ func main() {
 		r.Use(middleware.RealIP)
 		r.Use(middleware.Logger)
 		r.Use(middleware.Recoverer)
+		r.Use(cors.Handler(cors.Options{
+			AllowedOrigins:   []string{"https://*", "http://*"},
```

Comment:
Using the star is a ***very*** bad idea, especially when our API will not be public. I'll let it slide this time, because this is just a prototype, but please add new issue related to this immediately after the merge

Reply:
Well, I don't really know the target address of the frontend, so it was the simplest way to solve it. In a real deployment, they would likely have entirely different addresses (not `localhost` atleast), so that would introduce new problems. It will be fixed though.